Library/Homebrew/livecheck/skip_conditions.rb

```diff
@@ -0,0 +1,228 @@
+# typed: true
+# frozen_string_literal: true
+
+require "livecheck/livecheck"
+
+module Homebrew
+  module Livecheck
+    # The `Livecheck::SkipConditions` module primarily contains methods that
+    # check for various formula/cask conditions where a check should be skipped.
+    #
+    # @api private
+    module SkipConditions
+      extend T::Sig
+
+      module_function
+
+      sig do
+        params(
+          formula_or_cask: T.any(Formula, Cask::Cask),
+          livecheckable:   T::Boolean,
+          full_name:       T::Boolean,
+          verbose:         T::Boolean,
+        ).returns(T.nilable(Hash))
+      end
+      def formula_or_cask_skip(formula_or_cask, livecheckable, full_name: false, verbose: false)
```

Comment:
```suggestion
      end
      
      def formula_or_cask_skip(formula_or_cask, livecheckable, full_name: false, verbose: false)
```

Reply:
From what I've seen, we never include a blank line between the Sorbet method signature (even when it's multiline) and the associated method definition. Am I missing something here?